app/views/homepages/index.html.erb

```diff
@@ -0,0 +1,86 @@
+<main>
+  <section class="spotlight">
+    <h2 class="spotlight__header">
+      <span class="spotlight__header--prefix">
+         "Media Spotlight" *::after
```

Comment:
I assume you were trying to do something with CSS here.  Since this is just HTML there's no special syntax here:

```suggestion
         Media Spotlight:
```